.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 Layout/ExtraSpacing:
   ForceEqualSignAlignment: true
 
-Layout/IndentHash:
+Layout/IndentFirstHashElement:
```

Comment:
This was done as part of the versions PR. But I'm fine with it being in its own PR.